JAYESHBATRA/Virtuo-Learn
# fix: Fixed video button footer #317 issue

# Description:
This PR addresses the issue where the website's footer was incorrectly overlapping the video content when users scrolled down the page. This behavior was causing a visual obstruction and negatively impacting the user experience.

Fixes #317 

# Issue Fixed:
- The footer now stays at the bottom of the page and does not cover any content.


# How to Test:
1. Navigate to the page with the video content.
2. Scroll down to the bottom of the page.
3. Observe that the footer no longer overlaps the video content and remains at the bottom of the page.

# Screenshots:
**Before**
-----
![image](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/166977453/3f11f1e0-641e-4ffd-becc-d74827bbaa20)

-----
**After**
-----
![image](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/166977453/e4f7ce00-2951-4fa9-8bd6-68b0104154fc)
-----
**Before**
-----
![image](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/166977453/0cb774d5-001a-4bd0-9930-59cf7fc5ffa7)
-----
**After**
![image](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/166977453/56f6dac1-eb71-4903-b309-e09426ee3b17)
-----

I have fixed the footer for all videos. 

By merging this PR, we will enhance the site's usability and ensure a seamless viewing experience for all users.




## Checklist 

- My code adheres to the established style guidelines of the project.
- I have included comments in areas that may be difficult to understand.
- My changes have not introduced any new warnings.
- I have conducted a self-review of my code.
